src/Parsers/IAST.cpp

```diff
@@ -165,11 +167,26 @@ size_t IAST::checkDepthImpl(size_t max_depth) const
     return res;
 }
 
-std::string IAST::formatForErrorMessage() const
+String IAST::formatWithHiddenSecrets(size_t max_length, bool one_line, bool hilite) const
```

Comment:
do we use `hilite`, or this is for future usage ?

Reply:
I remove this parameter.